crepr/crepr.py

```diff
@@ -17,7 +9,6 @@
 from types import MappingProxyType
 from types import ModuleType
 from typing import Annotated
-from typing import Optional
```

Comment:
Optional is needed for typer to make a [CLI argument optional](https://typer.tiangolo.com/tutorial/arguments/optional/#make-an-optional-cli-argument)